charts/victoria-metrics-agent/templates/deployment.yaml

```diff
@@ -11,6 +12,8 @@ metadata:
   {{- end }}
 spec:
   replicas: {{ .Values.replicaCount }}
+  strategy:
```

Comment:
please specify strategy if value is not empty

Reply:
i've added condition